src/core/controller/index.ts

```diff
@@ -109,9 +109,7 @@ export class Controller {
 	- https://github.com/microsoft/vscode-extension-samples/blob/main/webview-sample/src/extension.ts
 	*/
 	async dispose() {
-		this.outputChannel.appendLine("Disposing ClineProvider...")
```

Comment:
The `outputChannel` is closed upon closing VSCode and therefore it throws an error when when try to `.appendLine`.